src/main/java/com/faforever/client/chat/ChatUserItemController.java

```diff
@@ -151,8 +151,72 @@ public void initialize() {
     JavaFxUtil.bind(playerMapImage.visibleProperty(), Bindings.isNotNull(playerMapImage.imageProperty()));
 
     updateFormat();
+    addEventHandlersToPlayerMapImage();
+  }
+
+  private void initializeTooltips() {
+    userTooltip = new Tooltip();
+    usernameLabel.setTooltip(userTooltip);
+
+    avatarTooltip = new Tooltip();
+    avatarTooltip.setAnchorLocation(PopupWindow.AnchorLocation.CONTENT_TOP_LEFT);
+    Tooltip.install(avatarImageView, avatarTooltip);
+
+    countryTooltip = new Tooltip();
+    countryTooltip.showDelayProperty().set(Duration.millis(250));
+    Tooltip.install(countryImageView, countryTooltip);
+
+    statusGameTooltip = new Tooltip();
+    statusGameTooltip.setShowDuration(Duration.seconds(30));
+    statusGameTooltip.setShowDelay(Duration.ZERO);
+    statusGameTooltip.setHideDelay(Duration.ZERO);
+    Tooltip.install(playerStatusIndicator, statusGameTooltip);
   }
```

Comment:
Chat user items are in a recycler view. Meaning tons of them are created and destroyed while scrolling. So all this is safer if it runs on mouse over only

Reply:
This was a change in the chat optimization so might not be related to this pr specifically and would probably need to be done in another pr.

Reply:
I understand that there is no need to redo this code?

Reply:
Yeah if it is redone it wont be in this pr